### ISSUE TITLE
[BUG]: PHPStan is missing an input for `magento_pre_install_script`

### ISSUE THREAD

**Issue by damienwebdev** [extdn/github-actions-m2#41]

**Comment by jissereitsma** [comment]
Could you elaborate a bit on this bug? The PHPStan workflow is defining the input `magento_pre_install_script` here: https://github.com/extdn/github-actions-m2/blob/master/magento-phpstan/action.yml#L19 Next the input is used here: https://github.com/extdn/github-actions-m2/blob/master/magento-phpstan/entrypoint.sh#L31 

It doesn't give me any issues, either specifying that input or skipping it:
```yaml
name: ExtDN M2 PHPStan
on: [push, pull_request]

jobs:
  phpstan:
    name: M2 PHPStan
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v2
      - uses: extdn/github-actions-m2/magento-phpstan@master
        with:
          composer_name: ${{ secrets.COMPOSER_NAME }}
          magento_pre_install_script: .github/workflows/extdn-phpstan-pre-install.sh
```

What kind of issue are you having exactly?

**Comment by damienwebdev** [comment]
Your commit addressed my concern! <3 49d1acd5be8215e591e9c18f1e091cb931a39222

**Comment by jissereitsma** [comment]
:)
